data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -317,6 +317,15 @@ t_rp_publication_date AS (
   WHERE rn=1
 ),
 
+t_vor_publication_date AS (
+  SELECT 
+    article_id,
+    vor_publication_date
```

Comment:
I would expect `vor_publication_date` to be of type DATE.
I think we probably should update `v_elife_article_xml_data`.

Reply:
I have updated that view.